sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/models/AmqpBodyType.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.models;
+
+import java.util.Locale;
+
+/**
+ * All AmqpBodyType available for AMQP Message.
+ */
+public enum AmqpBodyType {
+    /**
+     * Represent Amqp Data type
+     */
+    DATA("Data"),
```

Comment:
Is it necessary to have `("Data")`? I think it would be just as informational to use .tostring on the enum value and get the value in all caps without adding complexity to this class..